otherlibs/stdune/src/io.ml

```diff
@@ -268,6 +268,18 @@ struct
       fun ?chmod ~src ~dst () ->
         let src = Path.to_string src in
         let dst = Path.to_string dst in
+        let src_stats =
+          match Unix.stat src with
+          | exception Unix.Unix_error (Unix.ENOENT, "realpath", _) ->
```

Comment:
The string isn't going to be `"realpath"` in this pattern. I think it would be better not to match on it at all

Reply:
good catch, this is an oversight.